WordPress/Classes/ViewRelated/Jetpack/Social/JetpackSocialNoConnectionView.swift

```diff
@@ -33,6 +34,7 @@ struct JetpackSocialNoConnectionView: View {
                 }
             }
         }
+        .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .topLeading)
```

Comment:
This was a fun one to figure out 🙃. It fixes the following issues:

1. The `UIHostingController`'s view had extra space causing issues with the insets. `maxWidth` & `maxHeight` here expand the SwiftUI view to the entire host's view.
2. Even with the view fully expanded now, it was being aligned to the center of the view. This also caused issues with the insets. Changing the alignment to `.topLeading` makes the insets work as you'd expect.

Reply:
Yeah, there's a lot of quirky behaviors for sure. I've also stumbled on this weird behavior and ended up seeing [this SO post](https://stackoverflow.com/questions/69265914/on-ios-15-the-uihostingcontroller-is-adding-some-weird-extra-padding-to-its-hos) (there's a comment from Salim, too! 🕵🏼 ) which mentions that it's fixed on iOS 16. But I guess you're testing on an iOS 16 Simulator? 🙃 

Anyways, glad you've found a workaround 👍🏼

Reply:
> But I guess you're testing on an iOS 16 Simulator?

Yep, my simulators are `16.4` so I'm not sure if it was fixed with iOS 16 or if this was a different issue.